src/murmur/Murmur.ice

```diff
@@ -485,7 +487,8 @@ module Murmur
 		 * @param key Configuration key.
 		 * @return Configuration value. If this is empty, see {@link Meta.getDefaultConf}
 		 */
-		idempotent string getConf(string key) throws InvalidSecretException;
+		idempotent string getConf(string key) throws InvalidSecretException,
+		WriteOnlyException;
```

Comment:
Is your editor tricking you? We don't indent because we don't put in any line breaks, from what I can see? Just keep WriteOnlyException on the previous line.